storm-server/src/main/java/org/apache/storm/scheduler/resource/ResourceUtils.java

```diff
@@ -57,7 +64,8 @@
         if (topology.get_spouts() != null) {
             for (Map.Entry<String, SpoutSpec> spout : topology.get_spouts().entrySet()) {
                 Map<String, Double> topologyResources = parseResources(spout.getValue().get_common().get_json_conf());
-                checkIntialization(topologyResources, spout.getValue().toString(), topologyConf);
+                checkInitialization(topologyResources, spout.getKey(), topologyConf);
+                LOG.warn("Turned {} into {}", spout.getValue().get_common().get_json_conf(), topologyResources);
```

Comment:
Can we delete this and the one on line 54 too?  it looks like a debug statement that accidentally slipped into the code. Truth be told I think it is my debug statement that slipped into this...